CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+# 0.4.0
+
+> Published 27 Aug 2025
+
+## Major Features
+
+- **Integration with Observability Tools**:
+  - **LangFuse Integration**: Span adapters for Langfuse client, including open telemetry and graph visualisation (KG-217, KG-223)
+  - **Weave Integration**: Span adapters for Weave open telemetry and observability (KG-217, KG-218)
+- **Ktor Integration**: First-class Ktor support via the "Koog" Ktor plugin to register and run agents in Ktor applications (#422).
+- **iOS Target Support**: Multiplatform expanded with native iOS targets, enabling agents to run on Apple platforms (#512).
+- **Structured Output 2.0**: Refactored structured output API to be more flexible and add built-in/native provider support for OpenAI and Google, reducing prompt boilerplate and improving validation (#443).
```

Comment:
Shall we call it `2.0`? Maybe just "reworked" or "upgraded" structured output?

Reply:
upgraded is better - wil change